frontend/src/pages/NewRunV2.tsx

```diff
@@ -72,11 +80,42 @@ function NewRunV2(props: NewRunV2Props) {
   const [isStartingNewRun, setIsStartingNewRun] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
   const [isParameterValid, setIsParameterValid] = useState(false);
+  const [clonedRuntimeConfig, setClonedRuntimeConfig] = useState<PipelineSpecRuntimeConfig>({});
+  const [usePipelineFromRunLabel, setUsePipelineFromRunLabel] = useState(
```

Comment:
I guess you don't need `setUsePipelineFromRunLabel` in this case? Does `npm run start` produce warning in this case? We need to make sure no warning is produced when running `npm run start`.